dali/operators/bbox/bb_flip_test.cc

```diff
@@ -100,9 +100,9 @@ const TestSample &FindSample(const TestSample (&dataset)[N], const Roi &roi) {
 template<size_t Idx, typename Backend, size_t N>
 std::unique_ptr<TensorList<Backend>> ToTensorList(const TestSample (&sample)[N]) {
   std::unique_ptr<TensorList<Backend>> tl(new TensorList<Backend>());
-  tl->Resize(uniform_list_shape(N, {kBbStructSize}));
-  auto ptr = tl->template mutable_data<float>();
   for (size_t n = 0; n < N; n++) {
+  tl->Resize(uniform_list_shape(N, {kBbStructSize}));
```

Comment:
Does this line need to be inside for loop?

Reply:
Moved accidentally, fixed.